core/atomic_sync.c

```diff
@@ -29,6 +29,11 @@
 #include <string.h>
 #include "irq.h"
 
+/*
+ * uncrustify mis-formats the macros in this file, so disable it globally.
+ * begin{code-style-ignore}
```

Comment:
`/* end{code-style-ignore} */` missing?

Reply:
Well, I could add it at the end. The whole file is supposed to be ignored.

Reply:
Even though it might not be checked, I think its cleaner to always have an end to a beginning ;-).